misk/src/main/kotlin/misk/web/interceptors/MarshallerInterceptor.kt

```diff
@@ -35,6 +36,9 @@ internal class MarshallerInterceptor constructor(private val marshaller: Marshal
     @JvmSuppressWildcards private val marshallerFactories: List<Marshaller.Factory>
   ) : Interceptor.Factory {
     override fun create(action: Action): Interceptor? {
+      if (action.returnType.classifier == WebSocketListener::class) {
+        return null
```

Comment:
short comment?

Reply:
The fact that we have to hard-code implicit type-based conditionals to short-circuit the normal request routing logic when web sockets are involved makes me think we shouldn't be running web socket connects through the action routing system.